058-PowerPlatformBasics/Coach/Solution-04.md

```diff
@@ -0,0 +1,19 @@
+# Challenge 04 - Automation - Approval Process - Coach's Guide 
+
+[< Previous Solution](./Solution-03.md) - **[Home](./README.md)** - [Next Solution >](./Solution-05.md)
+
+## Notes & Guidance
+
+This is the only section you need to include.
```

Comment:
remove boilerplate text

"This is the only section you need to include.

Use general non-bulleted text for the beginning of a solution area for this challenge"